search-enterprise/latest/upgrade/index.md

```diff
@@ -43,3 +43,69 @@ Your current Content Services stack can continue to run while you are indexing t
 ![upgraded-environment]({% link search-enterprise/images/elasticsearch-upgrading-2.png %})
 
 > **Note:** You may need to use the Elasticsearch Re-indexing application to update to the latest changes. After that, new and updated documents will be uploaded to the Elasticsearch index by the Elasticsearch connector service using ActiveMQ messages.
+
+## Zero downtime upgrade
+
+You can upgrade from Search Services 2.x without experiencing any downtime, to Search Enterprise 3.1.1 when you are using Content Services 7.2 and above.
+
+1. Start an Elasticsearch 3.1.1 instance, for more see [Overview]({% link search-enterprise/latest/install/index.md %}).
+
+    Currently your installation is using solr.
```

Comment:
Change solr to Solr in this page (there are several instances).

Reply:
Updated